pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
+> Provide a standardized way of manipulating a file/directory's ownership and permissions while copying file or creating directory.
```

Comment:
There's no such explanation. However, from the linguistics point-of-view says: place (something) in a new position of authority. But that has nothing to do with a man page, for example the 'cron command' https://github.com/tldr-pages/tldr/blob/master/pages/common/crontab.md didn't say that it named after "chronos" the Greek word for time.